### ISSUE TITLE
[v3.0.2.0] autocomplete limit - Edit Product > Links > Categories

### ISSUE THREAD

**Issue by toonetcreation** [opencart/opencart#6503]
Hi,

Regarding this issue : https://forum.opencart.com/viewtopic.php?t=140199
Is it possible to fix it in next release?
Because autocomplete limit is limited to 5 results by default and many customers have more than 5 categories or sub-categories .. so some results are hidden.

Thanks

**Comment by toonetcreation** [comment]
hello, any news? thanks

**Comment by AlexChanHK** [comment]
The case always happened if the user using categories with same name.
If the developer are not going to edit the limitation of auto complete,
the most fastest way is base on your requirement to modify that limitation yourself.
(otherwise, a guy want 10 and b guy want 20, it will be very busy)

**Comment by toonetcreation** [comment]
the problem is : to modify that limitation by myself, this mean changing core files.

the current behavior is not logic : normaly the system should retreive all existing categories and not only 5.

I'm sorry but this is stupid !

**Comment by AlexChanHK** [comment]
I' m sorry but this is the only way.
We provide the solution for you already, and you just need to edit the number only.
Opencart system is opensource, it will not fit all user's requirement as every one for different case.
Otherwise, you can try with some extension.
https://www.opencart.com/index.php?route=marketplace/extension/info&extension_id=11373

**Comment by toonetcreation** [comment]
Yes I understand perfectly you're right, this is an open source system and you cannot add always specific features.
But to be honest I don't understand why it could be logic/clever to retreive just 5 categories even if you have 100 categories on your shop....I don't understand the interest of this setting.
I don't think retreiving all available categories will be useful only for me...contrariwise I think this should be the normal behavior for all users to dislpay all items/categories available.
why a lambda user will be interested to see just 5 categories over 100 ???
If I have 100 categories on my shop, I need to see the 100 categories.
If I have 50 categories on my shop, I need to see the 50 categories.
If I have 10 categories on my shop, I need to see the 10 categories.
etc...
normally, the best way should be a dynamic setting to systematically display all available categories and not an arbitrary setting equal to 5.
sorry again, I love open source, open cart and all these kind of software but this example is not a good conception way.

**Comment by danijelGombac** [comment]
@toonetcreation 

I have more than 200 categories, but when i type the name of the 30th category, this appear in the list. Why should display all 200 categories and then search in list?

**Comment by toonetcreation** [comment]
you're right, but was not my case.

I have 265 categories and previously (before file modificaiton) when I was typing the 30th category only the 5 first categories were retreived.

So categories from 6 to 265 were hidden....

**Comment by AlexChanHK** [comment]
@toonetcreation 
Do you really understand what is the meaning auto complete!?
If you want to show all the categories,
then you need to customize the code to disable autocomplete and using drop down list to replace it.

All in all, it is not a issue of Opencart system.

**Comment by toonetcreation** [comment]
Yes I know what is auto complete.
But as I said, in my case was nor working.
only first 5 results were displayed .

**Comment by osworx** [comment]
@toonetcreation Don't cry here around!
You mentioned the forum thead, use my extension there posted and you will have more results.
Otherwise change by yourself or as for paid support/help > in the forum!

**Comment by toonetcreation** [comment]
I'm not crying.
Ok I will look this extension.

thanks
